apps/openmw/mwsound/soundmanagerimp.cpp

```diff
@@ -1021,7 +1021,7 @@ namespace MWSound
 
                     if(sound->getDistanceCull())
                     {
-                        if((mListenerPos - objpos).length2() > 2000*2000)
+                        if((mListenerPos - objpos).length2() > sfx->mMaxDist*sfx->mMaxDist)
```

Comment:
Are you sure this is fine? IIRC the original game uses 2000, regardless of what's in the sound record. See [bug 244 about loud ghostgate sounds](https://bugs.openmw.org/issues/244), and the comment next to the definition of Play_RemoveAtDistance.

Reply:
I was testing things around Seyda Neen and saw that sounds that were being culled had a max distance of 2000, so it seemed having the hardcoded cull distance at 2000 was unnecessary and potentially problematic if modded sounds wanted to be audible farther away.

In regards to Ghostgate, I checked and it doesn't seem to be obnoxiously loud like this. Although that may be due to OpenAL Soft now having a volume limiter (it reduces the overall gain when the output samples would clip). The video in bug 244 showcases what the sound is like in vanilla's software audio mode, which can have extra limitations due to needing to run on weaker CPUs. Unfortunately, there doesn't seem to be any examples of it in hardware/DirectSound3D mode for comparison.

I'll set the vanilla game up to run in Wine with emulated hardware audio to see how it behaves.

Reply:
For now I've reverted it back to a hardcoded 2000, because I ran into another place where sounds became unnecessarily loud ('dagoth ur, outer facility'). However, I ran into a separate issue with sound sources of projectiles sometimes getting "lost" (intermittently the sound sources for the projectiles don't stop when they're supposed to, but are no longer being tracked either). I need to figure out what's going on with that before I update the PR.